project.clj

```diff
@@ -14,9 +14,9 @@
   :plugins [[lein-cljsbuild "1.1.7"]
             [lein-re-frisk "0.5.8"]
             [lein-cljfmt "0.5.7"]
-            [lein-githooks "0.1.0"]]
+            [rasom/lein-githooks "0.1.3"]]
```

Comment:
Could we consider merging those changes upstream?

Reply:
@jeluard latest changes in that repo are 3 years old, i don't really want to bother about this tbh

Reply:
we can move it to `status-im/lein-githooks` though